### PR TITLE
Feature/refactor account screen

### PR DESCRIPTION
**Pull request by Perronef5**
## CHANGES 📲
- Refactor Account screen to match Petes Spec

## DEMO 📸
Simulator Screen Recording - iPhone 14 - 2023-01-04 at 13 50 24

### CODE REVIEW

File: src/components/ModalScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <Text variant="subtitle1" textAlign="center">
+          {title}
+        </Text>
+      </Box>
+      {children}
+    </Box>
+  )
+}
+
+export default ModalScreen
```
Comment (tyler-whitman) [inline src/components/ModalScreen.tsx:42]: could add memo()

Comment (tyler-whitman) [reply src/components/ModalScreen.tsx:42]: Also I dont think this is used?

File: src/components/TokenPricesTicker.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (tyler-whitman) [inline src/components/TokenPricesTicker.tsx:1]: Can remove

File: src/features/account/AccountBalanceChart.tsx [hunk 3]
```diff
@@ -49,9 +48,9 @@ const AccountBalanceChart = ({
   const processedColors = useMemo(
     () => ({
       primaryBackground: processColor(primaryBackground),
-      primaryText: processColor(primaryText),
+      primaryText: processColor('#8C8C8C'),
```
Comment (tyler-whitman) [inline src/features/account/AccountBalanceChart.tsx:51]: Should we pull this color from the theme? Or is it a one off?

File: src/features/account/AccountTokenList.tsx [hunk 4]
```diff
@@ -160,6 +169,16 @@ const AccountTokenList = ({ loading = false }: Props) => {
       renderItem={renderItem}
       ListEmptyComponent={renderFooter}
       keyExtractor={keyExtractor}
+      refreshing={refreshing}
+      onRefresh={onRefresh}
+      onLayout={onLayout}
+      refreshControl={
```
Comment (tyler-whitman) [inline src/features/account/AccountTokenList.tsx:175]: Nice, I like it

File: src/features/account/AccountView.tsx [hunk 5]
```diff
@@ -56,25 +60,6 @@ const AccountView = ({
...
-
-    if (!hntPrice && !selectedBalance) return
-
-    let price = hntPrice
-
-    if (selectedBalance) {
-      price = selectedBalance.hntPrice
-    }
-
-    CurrencyFormatter.format(price || 0, currency).then((p) =>
```
Comment (tyler-whitman) [inline src/features/account/AccountView.tsx:73]: Do we not need this anymore?

File: src/components/TokenPricesTicker.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+  const { currentData: tokenPrices } = useGetTokenPricesQuery(
+    { tokens: 'helium,solana', currency },
+    {
+      refetchOnMountOrArgChange: true,
+    },
+  )
+
+  const text = useMemo(() => {
+    if (!tokenPrices) return t('general.noData')
```
Comment (tyler-whitman) [inline src/components/TokenPricesTicker.tsx:33]: should be `accountView.noData`

**Review by tyler-whitman — APPROVED** [review]
Looks good, just some minor comments

File: src/components/ModalScreen.tsx (hunk 1 above)
Comment (Perronef5) [reply src/components/ModalScreen.tsx:42]: Hmm yeah its being used in the coinbase pay draft PR. Will remove

File: src/features/account/AccountBalanceChart.tsx (hunk 3 above)
Comment (Perronef5) [reply src/features/account/AccountBalanceChart.tsx:51]: Its a one off

File: src/features/account/AccountView.tsx (hunk 5 above)
Comment (Perronef5) [reply src/features/account/AccountView.tsx:73]: Nope this is all happening in the TokenPricesTicker now

**Pull request merged**